vufs/src/main/java/com/stnetix/ariaddna/vufs/DTO/ChunkFile.java

```diff
@@ -0,0 +1,29 @@
+package com.stnetix.ariaddna.vufs.DTO;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Describe the chunk of file. Field url contain the place where this chunk stored.
+ * Used in class {@link VirtualFile} as metadata of part of file.
+ */
+public class ChunkFile implements Serializable {
+    private Long id;
+    private List<String> urls;
+
```

Comment:
I suggest you to add here the third private filed which will be responsible for storing MD5 hash-sum of a chunk. It will allow us to control the data integrity while reconstructing the user's file at his new computer. 
I mean the following:
- we have got a chunk
- calculated an MD5 hash for it
- written it as a String (?) field and now keeping it
- at a new device a user will receive a chunk from a cloud
- an MD5 hash will be calculated and compared with that we are keeping
- are they the same ? attach the chunk to a file to be reconstructed : repeat download of a chunk from another cloud